sklearn/ensemble/_stacking.py

```diff
@@ -80,10 +80,17 @@ def _concatenate_predictions(self, X, predictions):
         X_meta = []
         for est_idx, preds in enumerate(predictions):
             # case where the the estimator returned a 1D array
-            if preds.ndim == 1:
+            if isinstance(preds, np.ndarray) and preds.ndim == 1:
                 X_meta.append(preds.reshape(-1, 1))
             else:
                 if (
+                    self.stack_method_[est_idx] == "predict_proba"
+                    and self._type_of_target == "multilabel-indicator"
+                    and isinstance(preds, list)
+                ):
```

Comment:
It would be good to put a comment to explain what type do we expect with `predict_proba` here.

Reply:
I assume that we get a list of NumPy array of a given shape.